py_bentoctl/other_helpers.py

```diff
@@ -271,6 +272,8 @@ def init_docker(client: docker.DockerClient):
         ("BENTO_NOTIFICATION_NETWORK", dict(driver="bridge")),
         ("BENTO_PUBLIC_NETWORK", dict(driver="bridge")),
         ("BENTO_REDIS_NETWORK", dict(driver="bridge", internal=True)),  # Does not need to access the web
+        ("BENTO_REFERENCE_NETWORK", dict(driver="bridge")),
+        ("BENTO_REFERENCE_DB_NETWORK", dict(driver="bridge", internal=True)),  # Does not need to access the web
```

Comment:
`docker-compose.yaml` sets this network as external, `lib/reference/docker-compose.reference.yaml` sets it as both external and internal

Reply:
these aren't technically opposites, but i do think setting internal in the compose files doesn't actually do anything since it's set when running `init-docker`